server/build/webpack.js

```diff
@@ -280,6 +280,7 @@ export default async function createCompiler (dir, { dev = false, quiet = false,
       }
     },
     resolve: {
+      extensions: ['', '.js', '.jsx', '.json'],
```

Comment:
Remove .jsx. We don't wanna support it. We have talked about this somewhere else.